contents/2_why_julia.md

```diff
@@ -42,21 +42,20 @@ Well, **in Julia Python or R minutes can be translated to seconds**^[and sometim
 We reserved @sec:julia_wild for displaying successful Julia usecases in both academia and industry.
 
 
-1. Tried to do something different than `numpy`/`dplyr` conventions and discovered that your code is slow and you'll probably have to learn dark magic^[`numba`, or even `Rcpp` or `cython`?] to make it faster?
-
+2. Tried to do something different than `numpy`/`dplyr` conventions and discovered that your code is slow and you'll probably have to learn dark magic^[`numba`, or even `Rcpp` or `cython`?] to make it faster?
 **In Julia you can do your custom different stuff without loss of performance**.
 
-2. Had to debug code and somehow you see yourself reading a Fortran or C/C++ source code and having no idea what you are trying to accomplish?
+3. Had to debug code and somehow you see yourself reading a Fortran or C/C++ source code and having no idea what you are trying to accomplish?
```

Comment:
You could also write `1.` for all the items and Pandoc will do the numbering for you.